Rocket.Chat/Models/AuthSettings.swift

```diff
@@ -81,6 +81,8 @@ final class AuthSettings: BaseModel {
     @objc dynamic var hideMessageUserRemoved: Bool = false
 
     // Message
+    @objc dynamic var messageAllowPinning = true
```

Comment:
@rafaelks Doesn't this naming pattern sound weird for you? 

"message allow pinning" <- Actually is not the message that allows or deny pinning, it is the server, right?

We could use `serverAllowsMessagePinning` and it would be more meaningful. Or we could even put it in the user perspective and name it `isAllowedToPinMessages` because even though is a server setting it is related to an action that the user performs. 

Does it make any sense?

Reply:
@filipealva I think we should follow the back-end naming "Message_AllowPinning"

Reply:
@cardoso Got it! 

I personally disagree with this, I rather follow the naming convention of each platform, also I think that part of the job of "mapping" it to our `.swift` model is to make it conform to our naming conventions. 

But if that's the pattern that we are adopting, no problems, but in this case I would say that using literally `Message_AllowPinning`  or `message_allowPinning` would be more meaningful for the developers to read.

Reply:
I agree that it's not the best naming. But It's not a big problem if we are consistent.

Probably the best way to do this would be to separate each namespace into a different struct with their own mapping functions. So we would access it like this:

```swift
settings.message.allowPinning
```